guillotina/api/files.py

```diff
@@ -110,6 +111,34 @@ class UploadFile(TraversableFieldService):
         return await adapter.upload()
 
 
+@configure.service(
+    context=IResource,
+    method="POST",
+    permission="guillotina.DeleteContent",
+    name="@delete/{field_name}",
+    parameters=[{"in": "path", "name": "field_name", "required": True, "schema": {"type": "string"}}],
+    **_traversed_file_doc("Update the content of a file"),
+)
+@configure.service(
+    context=IResource,
+    method="POST",
```

Comment:
post, why?
it should be DELETE

Reply:
I'm ok with changing to DELETE. I wasn't in this case because we are not deleting the entire resource, we are doing a PATCH (I guess is the same reason `@upload` is a PATCH instead of POST)

Reply:
Mmmm... The file is a resource child of...

Reply:
Well the file is stored in a field of the resource, is part of the resource. Anyway, I changed it to DELETE